hostports/watcher.go

```diff
@@ -324,3 +366,57 @@ func setupKernelParameters() error {
 	}
 	return nil
 }
+
+func getBridgeInfo(network metadata.Network, host metadata.Host) (bridge string, bridgeSubnet string) {
+	conf, _ := network.Metadata["cniConfig"].(map[string]interface{})
+	for _, file := range conf {
+		props, _ := file.(map[string]interface{})
+		cniType, _ := props["type"].(string)
+		checkBridge, _ := props["bridge"].(string)
+		bridgeSubnet, _ = props["bridgeSubnet"].(string)
+
+		hostSubnet, _ := host.Labels[hostSubnetLabel]
```

Comment:
The above comment that I made is not applicable anymore.

Use the logic from other PR 76: `subnet: __host_label__: io.rancher.network.per_host_subnet.subnet`